docs/source/user_guide/12_list_of_environment_variables.rst

```diff
@@ -0,0 +1,184 @@
+      - FALSE
+    * - .. envvar:: RADICAL_PILOT_STRICT_CANCEL
+      - Cancel the tasks immediately. #TODO Didn't understood
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_DEFAULT_LOG_TGT
+      - The log target. #TODO Need more clarification
+      - .
+    * - .. envvar:: RADICAL_DEFAULT_LOG_DIR
+      - The Log dir
+      - $PWD
+    * - .. envvar:: RADICAL_DEFAULT_LOG_LVL
+      - #TODO Description
+      - ERROR
+    * - .. envvar:: RADICAL_DEFAULT_REPORT
+      - #TODO Description
+      - TRUE
+    * - .. envvar:: RADICAL_DEFAULT_REPORT_TGT
+      - #TODO Description
+      - stderr
+    * - .. envvar:: RADICAL_DEFAULT_REPORT_DIR
+      - Directory of Reporter module
+      - $PWD
+    * - .. envvar:: RADICAL_DEFAULT_PROFILE
+      - #TODO Description
+      - TRUE
+    * - .. envvar:: RADICAL_DEFAULT_PROFILE_DIR
+      - #TODO Description
+      - $PWD
+
+.. raw:: html
+
+   <hr>
+
+.. list-table:: Logging-related Env vars.
+    `ru.Logger` instances have a name and a name space.
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - <NS>_LOG_LVL
+      - Used for log level. #TODO Description
+      - Refer `RADICAL_DEFAULT_*` #TODO Confirm these
+    * - <NS>_LOG_TGT
+      - Used for the log targets
+      - Refer `RADICAL_DEFAULT_*` #TODO Confirm these
+
+.. note:: The name space is used to derive env variable names for log levels and targets. If no ns is given, the ns is derived from the name. Eg. the name `radical.pilot` becomes `RADICAL_PILOT`.
+
+.. raw:: html
+
+   <hr>
+
+.. list-table:: Reporter-related Env vars. `ru.Reporter` instances are very similar to `ru.Logger` instances: same schema is used for names and name spaces.
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - <NS>_REPORT
+      - Boolean to turn on and off Reporter
+      - TRUE
+    * - <NS>_LOG_TGT
+      - Where to report to
+      - #TODO
+
+.. raw:: html
+
+   <hr>
+
+.. list-table:: Developer List
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - .. envvar:: RADICAL_UTILS_NO_ATFORK
+      - Monkeypatching can be disabled by setting RADICAL_UTILS_NO_ATFORK.
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_DEBUG
+      - enables scattered debug facilities. This will likely slow down and even destabilize the code.
+      - {NOT_SET}
+    * - RU_RAISE_ON_*
+      - related to :envvar:`RADICAL_DEBUG` to trigger specific exceptions
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_DEBUG_HELPER
+      - related to :envvar:`RADICAL_DEBUG`, enables a persistent debug helper class in the code and installs some signal handlers for extra debug output.
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_DEBUG_VERBOSE
+      - related to :envvar:`RADICAL_DEBUG`, enables verbose messages for debugging.
+      - #TODO
```

Comment:
Dont know the default value @mtitov

Reply:
Controls "debug" module to collect stack traces. Verbose flag sets the level of details for output messages.